src/components/dropdownSearch/DropdownSearch.tsx

```diff
@@ -28,7 +28,7 @@ export interface IDropdownSearchOwnProps extends React.ClassAttributes<DropdownS
   id: string;
   modMenu?: boolean;
   defaultOptions?: IDropdownOption[];
-  defaultSelectedOptions?: FixedQueue<IDropdownOption>;
+  defaultSelectedOption?: IDropdownOption;
```

Comment:
This is to make the DropdownSearch have a fake option by default that says something like: "Select option". That was how the component PO made worked. Maybe will we need to make this as a text placeholder instead